testsettings.py

```diff
@@ -163,3 +163,5 @@ def _set_logging_levels(levels):
     if "custom.icds.commcare_extensions" not in COMMCARE_EXTENSIONS:
         COMMCARE_EXTENSIONS.append("custom.icds.commcare_extensions")
         CUSTOM_DB_ROUTING["icds_reports"] = "icds-ucr-citus"
+
+FORMPLAYER_INTERNAL_AUTH_KEY = "abc123"
```

Comment:
I initially tried to set this on the same lines as https://github.com/dimagi/commcare-hq/blob/c59306c84fabe41cab0001829e4f288e86eafc9a/corehq/apps/ota/tests/test_formplayer_restore.py#L13 but the setting is not getting set and test fails with missing key. 
Spent quite some time but not sure why that is happening, so chose to fallback on setting this in testsettings.

Reply:
That seems fine, looks like this is already defined in dev_settings:
https://github.com/dimagi/commcare-hq/blob/49adb2934bff399f1c12a33baaa03018df6750ef/dev_settings.py#L120-L120

Reply:
Ya, but its not defined in testsetting (or even localsettings) which does not import devsettings. So it's only explicitly set in different settings files. So by this change we are now saying that there is an internal formplayer internal auth key in the test environment.
I think its a safe change and it effects only two tests (one i added now and one that was already present which overrides this setting) but just that it does effect the whole test environment so was trying to avoid it.

Reply:
It's defined in my localsettings, FWIW (that actually drove me a little crazy, wondering how this value was set).  I did try mocking this out, but it's challenging, since all the checks happen at module load time.  The setting can be easily overridden if [this check](https://github.com/dimagi/commcare-hq/blob/49adb2934bff399f1c12a33baaa03018df6750ef/corehq/util/hmac_request.py#L51-L51) is moved inside the `_inner` function - I'm not sure why it's pulled into that outer scope.

At any rate, I think it's fine to define this in testsettings, though it couldn't hurt to run that by whoever added it if you're concerned.  Having the setting present won't circumvent the auth checks (which _could_ render auth tests useless), as the request would still need to pass through the correct parameter.  The setting is only half of the equation.  It would, however, hide an error if code depends on that being present when it isn't (this is currently a soft failure).  To that end, this could be defined in `settings.py` with a dummy value, like how the django secret key is managed.

Reply:
Cool, thanks @esoergel 

> It's defined in my localsettings

I guess you needed it to run something (a case restore perhaps?) locally. Its not added by default (localsettings.example.py)

I did run this by Daniel but we could not conclude the conversation. Hence the second opinion.
I am going to go with this change.